JapaneseFilter/sections/specific.txt

```diff
@@ -41,6 +41,7 @@ spaia.jp,yomiuri.co.jp,fujinkoron.jp,mikle.jp,onayamifree.com,ytranking.net,doku
 kmansin03.com##.widget-img
 kmansin03.com##div[style^="width:100%; min-height:280px"]
 netkeiba.com##.Ad-main_footer
+liginc.co.jp##.info
```

Comment:
Okay but maybe it's safer to specify a bit more like `##.l-content-sub div.info`.